contracts/libraries/PlasmaVaultLib.sol

```diff
@@ -556,13 +556,22 @@ library PlasmaVaultLib {
     /// - Market-specific parameters
     /// - Fallback withdrawal routes
     function configureInstantWithdrawalFuses(InstantWithdrawalFusesParamsStruct[] calldata fuses_) internal {
+        // Get previous configuration to clean up stale entries
```

Comment:
as we discussed add index validation here getInstantWithdrawalFusesParams

Reply:
Now we have double security, memory cleanup, and read protection.